### ISSUE TITLE
TVM Conversion:AttributeError: 'String' object has no attribute 'array'

### ISSUE THREAD

**Issue by prashanthharshangi** [microsoft/hummingbird#530]
Ran into this issue on latest install of hummingbird. This happens when I load a saved tvm hummingbird model

Stacktrace: 
```
  File "/models/tvm_model.py", line 68, in __init__
    self.model = load(model_path)
  File "/Users/prashanth.harshangi/opt/anaconda3/envs/mlpipe/lib/python3.7/site-packages/hummingbird/ml/_utils.py", line 271, in load
    model = TVMSklearnContainer.load(location, do_unzip_and_model_type_check=False)
  File "/Users/prashanth.harshangi/opt/anaconda3/envs/mlpipe/lib/python3.7/site-packages/hummingbird/ml/containers/sklearn/tvm_containers.py", line 185, in load
    params = load_param_dict(open(os.path.join(location, constants.SAVE_LOAD_TVM_PARAMS_PATH), "rb").read())
  File "/Users/prashanth.harshangi/opt/anaconda3/envs/mlpipe/lib/python3.7/site-packages/hummingbird/ml/containers/sklearn/tvm_containers.py", line 147, in load_param_dict
    return {v.name: v.array for v in load_arr}
  File "/Users/prashanth.harshangi/opt/anaconda3/envs/mlpipe/lib/python3.7/site-packages/hummingbird/ml/containers/sklearn/tvm_containers.py", line 147, in <dictcomp>
    return {v.name: v.array for v in load_arr}
AttributeError: 'String' object has no attribute 'array'
```

Temporarily resolved by updating the param loader in tvm_container to :

```
        def load_param_dict(param_bytes):
            if isinstance(param_bytes, (bytes, str)):
                param_bytes = bytearray(param_bytes)
            load_arr = _load_param_dict(param_bytes)
            # return {v.name: v.array for v in load_arr}
            return {v[0]: np.array(v[1]) for v in load_arr}
```

**Comment by interesaaat** [comment]
Thanks for reporting this. I am looking into it.

**Comment by interesaaat** [comment]
Which version of TVM are you using? It looks that the version you are using has a different way of loading the saved parameters.

**Comment by prashanthharshangi** [comment]
Thanks for looking into it. I am using the nightly version of tvm from `https://tlcpack.ai/wheels` 
```
Name: tlcpack
Version: 0.8.dev107+g7b11b9217
```

**Comment by interesaaat** [comment]
Ok I think this is the problem. We only support releases and currently we are on TVM 0.7. We can fix this once they cut the 0.8 release. At least you are not blocked 😄 .
